src/jimple-frontend/AST/jimple_method_body.h

```
@@ -0,0 +1,116 @@
+//
+// Created by rafaelsamenezes on 21/09/2021.
+//
+
+#ifndef ESBMC_JIMPLE_METHOD_BODY_H
+#define ESBMC_JIMPLE_METHOD_BODY_H
+
+#include <jimple-frontend/AST/jimple_ast.h>
+#include <util/std_code.h>
+
+/**
+ * @brief A Jimple method definition
+ * 
+ * Something such as: public void foo() { }
+ */
+class jimple_method_body : public jimple_ast
+{
+public:
+  virtual exprt
+  to_exprt(contextt &, const std::string &, const std::string &) const
+  {
+    exprt dummy;
+    return dummy;
+  }
+};
+
+/**
+ * @brief The contents of a Jimple method
+ *
+ * This can be statements or declarations
+ */
+class jimple_method_field : public jimple_ast
+{
+public:
+  virtual exprt
+  to_exprt(contextt &, const std::string &, const std::string &) const
+  {
+    dump();
+    code_skipt dummy;
+    return dummy;
```

This seems to be used for debugging. Could this function assert() or even abort() rather than returning an empty expression?

---

I removed the dump